test/iron-overlay-behavior.html

```diff
@@ -133,14 +141,23 @@
           assert.isFalse(spy.called, 'overlay should not refit');
         });
 
-        test('open() triggers iron-resize', function(done) {
+        test('open() & close() trigger iron-resize', function(done) {
           // Ignore iron-resize triggered by window resize.
           var callCount = 0;
           window.addEventListener('resize', function() { callCount--; }, true);
           overlay.addEventListener('iron-resize', function() { callCount++; });
-          runAfterOpen(overlay, function() {
-            assert.equal(callCount, 1, 'iron-resize should be called once');
-            done();
+          overlay.open();
+          listenOnce(overlay, 'iron-overlay-opened', function() {
```

Comment:
listeners all the way downnn. 🐢 

Would this be clearer if it was just 2 tests? `open() triggers`, `close() triggers`